HW3/task1.py

```diff
@@ -0,0 +1,84 @@
+
+# 1. Використовуючи функцію print, надрукувати фразу “I love Python” 42 рази.
+# Оскільки цикли ще не розглядали - буде так:)
+print("I love Python")
```

Comment:
multiply operation should be used here
```suggestion
print("I love Python" * 42)
```